masonry/src/event_loop_runner.rs

```diff
@@ -51,11 +54,13 @@ pub enum WindowState<'a> {
     },
 }
 
-struct MainState<'a> {
+/// The state of the Masonry application. If you run masonry from an external event loop, create a
+/// MasonryState via MasonryState::new and forward winit events to it via the appropriate method (e.g.,
+/// masonry_state.handle_window_event via window_event on the winit ApplicationHandler).
```

Comment:
```suggestion
/// The state of the Masonry application. If you run Masonry from an external Winit event loop, create a
/// `MasonryState` via [`MasonryState::new`] and forward events to it via the appropriate method (e.g.,
/// calling [`handle_window_event`](MasonryState::handle_window_event) in [`window_event`](ApplicationHandler::window_event)).
```
Makes the doc comments follow the expected patterns